src/tests.yaml

```diff
@@ -0,0 +1,65 @@
+scenarios:
+- name: objectIds handles multiple ids
+  code: |-
+    const copyFromWindow = require('copyFromWindow');
```

Comment:
Would you consider making the build.js do a tiny parse of the YAML and "include" the test file? That way you could write the test in a foo.js file. If I had to write a test in a future PR I think I'd probably be a little turned off having to write it in YAML and may just decide it's too much effort.

Reply:
In the future, yes. I was looking for a minimal time investment here. What I'd really want is to be able to run these locally instead of having to do it all through the GTM console.